src/data/calculate.ts

```diff
@@ -237,16 +243,42 @@ export const calculatePersonRiskEach = (
       // If risk profile isn't selected, call it incomplete
       return null
     }
+
     const isHousemate =
       data.interaction === 'partner' || data.interaction === 'repeated'
-    return (
+    const unadjustedRisk =
       averagePersonRisk *
       personRiskMultiplier({
         riskProfile: RiskProfile[data.riskProfile],
         isHousemate,
         symptomsChecked: data.symptomsChecked,
       })
-    )
+
+    if (
+      // TODO: Support vaccinated risk for other Risk Profiles.
+      data.riskProfile !== 'average' ||
+      data.unvaccinatedPrevalenceRatio === null ||
+      data.averageFullyVaccinatedMultiplier === null
+    ) {
+      return unadjustedRisk
```

Comment:
Obviously this is beyond the scope of this PR, but we should make sure to indicate to people whether or not we're accounting for their vaccine. (Or only allow them to select it if we can account for it. And if we can't yet account for it, we should let them know why the field isn't showing up as an option with some risk profiles but is with others.)